cli/types/cypress.d.ts

```diff
@@ -5480,6 +5480,7 @@ declare namespace Cypress {
     extensions: string[]
     preferences: { [key: string]: any }
     args: string[]
+    env: { [key: string]: any }
```

Comment:
This is essentially a no-op in Electron, should we add an error or warning if someone sets .env with Electron?

Reply:
👍 I noticed that `args` seems to be the same so I've added a warning for both of those. Let me know if I've misread and `args` is used or if there's a different way that is preferred for warning.